### ISSUE TITLE
Are we sure we want to_json vs encode_json?

### ISSUE THREAD

**Issue by ambs** [PerlDancer/Dancer2#900]
From the documentation of JSON module, `to_json` keyword:

````````
If you want to write a modern perl code which communicates to outer world, you should use encode_json (supposed that JSON data are encoded in UTF-8).
````````
and I can't foresee a non UTF-8 world out there.

**Comment by logust79** [comment]
I don’t know if this is relevant, but there are many Chinese websites with non UTF-8 encoding.

**Comment by ambs** [comment]
Also, after digging it up a little, I found the utf8 flag on to_json 
disturbing. Needed to turn it off to output UTF-8 data.

Something is not clear in the way these modules work. At least for me.

**Comment by cromedome** [comment]
Closed by PR #1212. Thanks @nunorc!
